src/frontend/components/SøknadsSteg/Dokumentasjon/LastOppVedlegg.tsx

```diff
@@ -120,6 +121,9 @@ const LastOppVedlegg: React.FC<Props> = ({ dokumentasjon, vedleggNr }) => {
             <br />
             <Checkbox
                 label={<SpråkTekst id={'dokumentasjon.har-sendt-inn.spm'} />}
+                aria-label={`${formatMessage({
+                    id: 'dokumentasjon.har-sendt-inn.spm',
+                })} (${dokTittel})`}
```

Comment:
Hvorfor legger vi på doktittel her? :) bare nysgjerrig altså, men bare ser for meg at det kan være mye for brukeren hvis doktittel leses to ganger.

Reply:
Advarsel om at man ikke skal bruke samme label flere steder på siden, maket sense for meg at det kunne være forvirrende men jeg kan godt ta det vekk igjen hvis vi tror context holder.

Reply:
Ah, skjønner. Vi kan jo merge det inn, også får vi sikkert tilbakemelding når vi brukertester i morgen om hva det eventuelt burde være?

Reply:
:+1: